index.html

```diff
@@ -102,95 +110,129 @@ <h2>SIMPLE AND BEAUTIFUL</h2>
+                    </figure>
+                    <figure class="contact__container-items">
+                        <div class="contact__picture-second"></div>
                         <figcaption>
                             <p>CEO</p>
                             <p class="contact__paragraph">
                                 Roll Over Beethoven
                             </p>
                             <p>The one that puts it all together</p>
                             <div class="contact__picture-logo">
-                                <img src="./images/team-facebook.svg" alt="" />
-                                <img src="./images/team-twitter.svg" alt="" />
+                                <a href="https://pl-pl.facebook.com/">
+                                    <img
+                                        src="./images/team-facebook.svg"
+                                        alt="Logo-Facebook"
+                                /></a>
+                                <a href="https://twitter.com/?lang=pl"
+                                    ><img
+                                        src="./images/team-twitter.svg"
+                                        alt="Logo-Twitter"
+                                /></a>
                             </div>
                         </figcaption>
-                    </div>
-                    <div class="contact__container-items">
-                        <figure class="contact__picture-third"></figure>
+                    </figure>
+                    <figure class="contact__container-items">
+                        <div class="contact__picture-third"></div>
                         <figcaption>
                             <p>CFO</p>
                             <p class="contact__paragraph">Chuck Berry</p>
                             <p>The guy with his hand on the wallet</p>
                             <div class="contact__picture-logo">
-                                <img src="./images/team-facebook.svg" alt="" />
-                                <img src="./images/team-twitter.svg" alt="" />
+                                <a href="https://pl-pl.facebook.com/"
+                                    ><img
+                                        src="./images/team-facebook.svg"
+                                        alt="Logo-Facebook"
+                                /></a>
+                                <a href="https://twitter.com/?lang=pl"
+                                    ><img
+                                        src="./images/team-twitter.svg"
+                                        alt="Logo-Twitter"
+                                /></a>
                             </div>
                         </figcaption>
-                    </div>
+                    </figure>
                 </div>
-            </section>
+            </header>
             <footer class="footer" id="footer">
-                <p>STOPKA</p>
+                <p></p>
```

Comment:
Dlaczego nie ma stopki? :P

![image](https://github.com/devmentor-pl/task-html-and-css-basics/assets/55095257/4c1e6d46-3869-4e64-8d7c-bcbccff80f70)